### PR TITLE
Added test for non super user token

### PR DESCRIPTION
**Pull request by hasnain095**
I've created the test for a non-superuser token, and I've added a fixture that adds a test user and one that gets the user's bearer token.

### CODE REVIEW

**Comment by hasnain095** [comment]
#77 Created a pull request for this issue. I'll be glad to help manage this repo.

File: users/tests/conftest.py [hunk 1]
```diff
@@ -53,3 +56,28 @@ async def superuser_token_headers(client: AsyncClient) -> Dict[str, str]:
...
+async def create_non_superuser(session: AsyncSession) -> Dict[str, str]:
+    email = "test_user@test.com"
+    password = "Ksd8nASD1_Hjns!P"
+    hashed_password = get_password_hash(password)
+    result = await session.execute(select(User).where(User.email == email))
+    user: Optional[User] = result.scalars().first()
+    if user is None:
+        session.add(User(email=email, hashed_password=hashed_password, is_superuser=False))
+        await session.commit()
+    return {"email": email, "password": password}
```
Comment (Kludex) [inline users/tests/conftest.py:71]: Can this be a `scope="session"` instead?

**Comment by Kludex** [comment]
It looks like the pipeline is not running for PRs? :thinking:

**Comment by hasnain095** [comment]
I can take a look at the pipeline

**Comment by Kludex** [comment]
Any reason for closing this? :thinking:

**Comment by hasnain095** [comment]
I was eating dinner and working at the same time :), accidentally pushed some unwanted changes, and tried to revert using 'git reset' and that closed this PR. Kindy reopen it :)

**Comment by Kludex** [comment]
I'm not able to reopen from here. :sweat_smile:

**Comment by hasnain095** [comment]
Sorry about this, I've reopened it.

**Comment by Kludex** [comment]
There are no changes here :eyes:

**Pull request closed without merging**